### ISSUE TITLE
Which scheme to use?

### ISSUE THREAD

**Issue by jan-ivar** [w3c/webtransport#198]
(Broken out from https://github.com/w3c/webtransport/issues/183)

2. Should the scheme be wt: or https? If WebTransport is conceptually a framework that supports multiple protocols and each protocol is defined by its scheme, then wt: seems a poor choice as it conflates the framework name with the protocol descriptor. The exception here would be if the application cannot choose the underlying protocol being used in which case a wt: scheme would seem appropriate. Issue 180 seems relevant here.

**Comment by ricea** [comment]
My preference is for `https:` (with username, password and fragment disallowed). This provides the smoothest integration with the rest of the platform. 

I can foresee a few potential problems: parts of the platform (standards, implementation, tools or extensions) might assume that "https" means that fallback to HTTP/2 and HTTP/1 is supported, that CORS will happen, that cache headers will take effect, etc.

It might also surprise developers that things like cookies and HTTP authentication don't apply.

I don't like the idea of using a generic scheme like `wt:` because it will confuse matters if we need to differentiate by scheme in future.

A novel scheme like `http3-transport:` is a possibility but in my view it doesn't provide enough value to justify giving people another thing to remember.

**Comment by wilaw** [comment]
@aboba  Is this not an IETF decision? Can we ask for this issue to be resolved at IETF 110 meeting on March 8-12 before we issue first working draft?

**Comment by aboba** [comment]
Yes, it would be good to put this on the IETF 110 agenda. You can cover that (and any other relevant W3C issues) during the "W3C WebTransport Update" portion of the agenda.

**Comment by jan-ivar** [comment]
Meeting:
- Group inclined to prefer `https`

**Comment by aboba** [comment]
IETF 110 WEBTRANS meeting also favored `https`.
